### ISSUE TITLE
feat: Site Backgrounds and Images

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/spmssite#34]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/gbowne1/spmssite/blob/main/.github/CONTRIBUTING.md)
- [X] I agree to follow the [Code of Conduct]().
- [X] I have searched for [existing issues]() that already include this feature request, without success.

### Describe the Feature Request

I would suggest using for the background {between the header and footer} one of these:

https://www.google.com/search?q=brush+finish+metal+backround

so as long as the image is not copyrighted.

### Describe the Use Case

Website background.

**Comment by gbowne1** [comment]
Here are some typical images to add in the quality page.. put them in a /assets/ folder/directory

ISO 9001
https://www.google.com/search?source=univ&tbm=isch&q=iso+9001+images

AS9100 
https://www.google.com/search?q=as9100+images

Also add: 
- BBB image to Footer

**Comment by wyenelle** [comment]
assign me to this issue.

is it just to add a background image to your website?

**Comment by gbowne1** [comment]
No, this particular issue is to add typical images to the quality page (which should ideally be quality.html)

Other than that.. I would assume the best thing to do would be to add a light/dark mode theme with the colors:

So.. I would add a theme.css and use the switch from about.html 

Good theme colors are:
#0000ff blue
#000000 black
#808080 gray
#ffffff white
#c0c0c0 silver

The header has been blue and the navbar was black in the original site.  My original background was a brush finish steel or aluminum metallic look to it. something like..

![background](https://user-images.githubusercontent.com/47549872/186532030-3449b536-a9d1-4bdd-855a-ef0cf2cf50dc.jpg)
